### ISSUE TITLE
❗❗❗ Downtime 4/24-5/2 2023

### ISSUE THREAD

**Issue by kmoad** [KarchinLab/open-cravat#151]
**The datacenter hosting run.opencravat.org and the OpenCRAVAT module store will be down from Monday 4/24 to Monday 5/1 for maintenance on their cooling system.**

While run.opencravat.org will be unavailable during that window, users with local installations can continue working by configuring their system to [download modules from the AWS OpenData mirror](https://open-cravat.readthedocs.io/en/latest/AWS-OpenData.html).

Additionally, you can download finished jobs from run.opencravat.org to view locally.

- Click the row in the jobs table
- Click the "DB" button to download the job sqlite result
- On a system with OpenCRAVAT installed, run `oc gui result-file-name.sqlite`

We apologize for the inconvenience this causes. Further updates will be posted to this thread and to [our twitter](https://twitter.com/OpenCRAVAT).

**Comment by katelynmullen** [comment]
I cannot run OC from the command line yet. It successfully runs the converter, but then hangs indefinitely when running the gene mapper. Can you replicate this on your end? I am trying to run a tiny file of only 3 lines. 

[s_MPAM18_qcf_cravat.xlsx](https://github.com/KarchinLab/open-cravat/files/11383830/s_MPAM18_qcf_cravat.xlsx)

**Comment by skanderson** [comment]
@katelynmullen 

I was not able to replicate this problem just now; I saved that file as a tsv, then I ran `oc run s_MPAM18_qcf_cravat.tsv -a clinvar -l hg38` with no errors. 

Are you seeing files named `s_MPAM18_qcf_cravat.xlsx.err` and `s_MPAM18_qcf_cravat.xlsx.log` in the directory you ran the job? The contents of those might help us find the issue.

You might also want to run `oc module update` to make sure all your modules are up to date.

**Comment by kmoad** [comment]
It's run ok on my end as well. You could try reinstalling mapper. Also, what version of oc? Run `oc version`.

To reinstall mapper
```
oc module uninstall hg38
oc module install hg38
```

**Comment by katelynmullen** [comment]
I tried reinstalling mapper and oc in general and neither worked. I tried a test again this morning with the one patient I sent you and it successfully ran in 43 seconds. However, when I tried to cat a file of commands for 88 patients, gene mapper stalled. Then I tried to rerun the single patient again, but the gene mapper continued to stall.
![image](https://github.com/KarchinLab/open-cravat/assets/55409859/92c902df-985a-4959-927d-17d02ba32d5b)
